dist/s_string.ok

```diff
@@ -1242,6 +1242,7 @@ rle
 rmw
 rng
 rocksdb
+rollbacked
```

Comment:
The correct usage would be 'rolled back'. Can we please remove this and use that in the stat?

Reply:
Thanks, fixed.